### PR TITLE
blog post

### PR DESCRIPTION
**Pull request by michaelstaib**

### CODE REVIEW

File: src/content/blog/2023-08-25-graphql-fusion.md [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+
+![Simple Deployment Pipeline](/img/blog/2023-08-25-graphql-fusion/pipeline-1.png)
+
+## Apollo Federation
+
+We recognize that some of you may have opted into Apollo Federation as a solution, and that is why we designed the Fusion schema composition so that it can compose any Apollo Federation subgraph into a Fusion subgraph. This allows you to seamlessly migrate from an Apollo Federation setup to a GraphQL-Fusion setup without the need to rewrite anything.
+
+## Conclusion
+
+We are still working on GraphQL-Fusion, but you can try an early version already today with the Hot Chocolate Fusion Gateway. We plan to release the GraphQL-Fusion spec as it matures later this year under the **MIT license**. While the current iterations are primarily concerned with GraphQL, as mentioned before The Guild has begun specifying the Open API to GraphQL transformation they've developed in Mesh, and we'll integrate this as it becomes available. Hasura us working on GraphQL Compliant Data APIs spec and WunderGraph wants to contribute a spec for gRPC and Kafka (AsyncApi) to take federated GraphQL to a whole new level.
```
Comment (Urigo) [inline src/content/blog/2023-08-25-graphql-fusion.md]:
```suggestion
We are still working on GraphQL-Fusion, but you can try an early version already today with the Hot Chocolate Fusion Gateway. We plan to release the GraphQL-Fusion spec as it matures later this year under the GraphQL Foundation and as **MIT license**. While the current iterations are primarily concerned with GraphQL, as mentioned before The Guild, in collaboration with IBM and StepZen, has begun specifying the Open API to GraphQL transformation they've developed in Mesh, and we'll integrate this as it becomes available. Hasura is working on GraphQL Compliant Data APIs spec and WunderGraph wants to contribute a spec for gRPC and Kafka (AsyncApi) to take federated GraphQL to a whole new level.
```

**Review by Urigo — APPROVED** [review]

File: src/content/blog/2023-08-25-graphql-fusion.md [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+
+## Let's share and compete
+
+Late last year, [ChilliCream](https://chillicream.com) and [The Guild](https://the-guild.dev/) met in Paris and discussed their approaches towards distributed GraphQL. It became clear that both companies were solving similar problems, and we decided to join forces on this project. ChilliCream would provide the initial work on the Fusion spec and implementation. At the same time, The Guild would start specifying their work on [GraphQL Mesh Gateway](https://the-guild.dev/graphql/mesh) with [OpenAPI support](https://the-guild.dev/graphql/mesh/docs/handlers/openapi) and help shape the initial Fusion spec. As we started, work on prototypes and the initial spect texts, we reached out to more companies in the community to see if there was interest in collaboration. It turns out that the GraphQL community is hungry for an open specification to standardize distributed GraphQL application gateways. [Hasura](https://hasura.io/), [IBM](https://www.ibm.com/de-de), [solo.io](https://www.solo.io/), [AWS AppSync](https://aws.amazon.com/de/appsync/), [WunderGraph](https://wundergraph.com/) have all joined the effort for creating a common spec.
```
Comment (royderks) [inline src/content/blog/2023-08-25-graphql-fusion.md]: Link for IBM has localisation in it, this can be deleted :)

**Comment by leebyron** [comment]
Hey all, this is super exciting! It sounds from the content that this expected to be announced as a foundation-led effort, so let's nail down contribution paperwork ahead. I'll ask @jorydotcom to help out

File: src/content/blog/2023-08-25-graphql-fusion.md (hunk 2 above)
Comment (michaelstaib) [reply src/content/blog/2023-08-25-graphql-fusion.md]: This is now fixed.

**Comment by michaelstaib** [comment]
This blog post was superseded by the composite schema blog post and the work we have done around the composite schema spec.

**Pull request closed without merging**